.travis.yml

```diff
@@ -32,7 +32,6 @@ before_install:
     - npm --version
 install:
     - scripts/InstallPythonRequirements.py --mode=dev --ignore-plugins=${IGNORE_PLUGINS}
-    - npm install grunt grunt-cli
```

Comment:
While https://github.com/girder/girder/pull/973/files#r39280145 mention there is no need to specify `./node_modules/.bin/`,  removing the explicit install makes sense. `grunt ` and `grunt-cli` are specified as dependencies in `package.json` and will be automatically installed when calling `npm install`